app/models/manageiq/providers/openstack/infra_manager/refresh_parser.rb

```diff
@@ -365,6 +408,16 @@ def process_collection(collection, key)
           @data_index.store_path(key, uid, new_result)
         end
       end
+
+      def process_collection_item(item, key)
```

Comment:
ok, moving all to ManageIQ::Providers::Openstack::RefreshParserCommon::HelperMethods